src/app/units/states/tasks/inbox/directives/staff-task-list/staff-task-list.coffee

```diff
@@ -37,13 +37,6 @@ angular.module('doubtfire.units.states.tasks.inbox.directives.staff-task-list',
       filteredTasks = $filter('tasksInTutorials')(filteredTasks, $scope.filters.tutorials)
       filteredTasks = $filter('tasksWithStudentName')(filteredTasks, $scope.filters.studentName)
       $scope.filteredTasks = filteredTasks
-
-      # Fix selected task.
```

Comment:
Would be good to know what doesn't work here. This would be good to keep if we can.

Reply:
I've added a todo with more details